charts/prometheus-adapter/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus-adapter
-version: 4.9.1
+version: 4.9.2
```

Should be a minor bump since it's a new feature.

```suggestion
version: 4.10.0
```

---

should be fixed. forgot the DOC sign at first 😅